dub.sdl

```diff
@@ -3,9 +3,11 @@ description "libclang bindings / wrappers for D"
 authors "Atila Neves"
 copyright "Copyright © 2018, Atila Neves"
 license "BSD 3-clause"
-libs "clang"
+libs "clang" platform="posix"
+libs "libclang" platform="windows"
 lflags "-L/usr/local/opt/llvm/lib" platform="osx"
 lflags "-L/usr/lib/llvm-3.9/lib" platform="posix"  # travis
+lflags "/LIBPATH:C:\\PROGRA~1\\LLVM\\lib" platform="windows"
 
```

Comment:
`PROGRA~1` looks silly, but I found no other way to make spaces in `lflags` work.